_includes/releases/v22.1/v22.1.0-beta.1.md

```diff
@@ -6,90 +6,75 @@ Release Date: April 4, 2022
-- Adds a `crdb_internal.repair_ttl_table_scheduled_job` builtin, which repairs the given TTL table's scheduled job by supplanting it with a valid schedule. [#78373][#78373]
+ `ALTER SUPER REGION` follows the same rules as `ADD` or `DROP` super region. [#78462][#78462]
+
+- The [session variables](../v22.1/set-vars.html) `datestyle_enabled` and `intervalstyle_enabled`, and the [cluster settings](../v22.1/cluster-settings.html) `sql.defaults.datestyle.enabled` and `sql.defaults.intervalstyle.enabled` no longer have any effect. After upgrading to v22.1, these settings are effectively always interpreted as `true`. [#78455][#78455]
+- Bucket-count of [hash-sharded index](../v22.1/hash-sharded-indexes.html) is now shown from the `crdb_internal.table_indexes` table. [#78625][#78625]
+- Implemented the [`COPY FROM ... ESCAPE ...`](../v22.1/copy-from.html) syntax. [#78417][#78417]
+- Disabled index recommendations in [`EXPLAIN`](../v22.1/explain.html) output for [`REGIONAL BY ROW` tables](../v22.1/multiregion-overview.html#regional-by-row-tables), as the previous recommendations were not valid. [#78676][#78676]
+- Added a `crdb_internal.validate_ttl_scheduled_jobs` builtin. This verifies that each table points to a valid scheduled job which will action the deletion of expired rows. [#78373][#78373]
+- Added a `crdb_internal.repair_ttl_table_scheduled_job` builtin, which repairs the given TTL table's scheduled job by supplanting it with a valid schedule. [#78373][#78373]
 - `ALTER TABLE ... INJECT STATISTICS ...` will now issue notices when the given statistics JSON includes non-existent columns, rather than resulting in an error. Any statistics in the JSON for existing columns will be injected successfully. [#79071][#79071]
 
 <h3 id="v22-1-0-beta-1-operational-changes">Operational changes</h3>
 
-- This patch adds a new metric that charts the number of bytes received via snapshot on any given store. [#78464][#78464]
-- Bulk ingest operations like `IMPORT`, `RESTORE` or `CREATE INDEX` will now fail if they try to write to a node that has less than 5% storage capacity remaining, configurable via the setting `kv.bulk_io_write.min_capacity_remaining_fraction`. [#78579][#78579]
-- `IMPORT` jobs will now pause if a node runs out of disk space. [#78587][#78587]
-- `CREATE INDEX` and some other schema changes will now pause if a node is running out of disk space. [#78587][#78587]
-- `RESTORE` will now pause if a node is running out of disk space. [#78587][#78587]
+- Added a new metric that charts the number of bytes received via snapshot on any given store. [#78464][#78464]
+- Bulk ingest operations like [`IMPORT`](../v22.1/import.html), [`RESTORE`](../v22.1/restore.html) or [`CREATE INDEX](../v22.1/create-index.html)` will now fail if they try to write to a node that has less than 5% storage capacity remaining, configurable via the [`kv.bulk_io_write.min_capacity_remaining_fraction`](../v22.1/cluster-settings.html). [#78579][#78579]
```

Comment:
At `CREATE INDEX` ... the second back tick is missing before the square bracket.